app/models/block.rb

```diff
@@ -25,21 +25,12 @@ class Block < ApplicationRecord
   attribute :proposals, :ckb_array_hash, hash_length: ENV["DEFAULT_SHORT_HASH_LENGTH"]
 
   scope :recent, -> { order(timestamp: :desc) }
-  scope :available, -> { where(status: [:inauthentic, :authentic]) }
+  scope :accepted, -> { where(status: :accepted) }
```

Comment:
ActiveRecord provides scope based on the values of the enum field, we can remove this line.

Reply:
OK，I will fix this.